hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/queries/contract/HapiGetContractInfo.java

```diff
@@ -244,12 +242,7 @@ private Query getContractInfoQuery(HapiSpec spec, Transaction payment, boolean c
         } else {
             final var builder = ContractGetInfoQuery.newBuilder()
                     .setHeader(costOnly ? answerCostHeader(payment) : answerHeader(payment));
-            if (contract.length() == 40) {
-                builder.setContractID(
-                        ContractID.newBuilder().setEvmAddress(ByteString.copyFrom(CommonUtils.unhex(contract))));
-            } else {
-                builder.setContractID(asContractId(contract, spec));
-            }
+            builder.setContractID(asContractId(contract, spec));
```

Comment:
Removed the code above as the `asContractId` method has the same check for lenght and builds the same ContractID from the contractName/evmAddress